packages/react-native/lib/rn-rpc-server/async-storage-service.js

```diff
@@ -1,11 +1,11 @@
 import AsyncStorage from '@react-native-async-storage/async-storage';
 
 export class StorageService {
-  rpcMethods = [
-    StorageService.prototype.setItem,
-    StorageService.prototype.getItem,
-    StorageService.prototype.removeItem,
-  ];
+  rpcMethods = {
+    setItem: StorageService.prototype.setItem,
```

Comment:
I've updated the react-native services to provide the method names, so that js minifiers will not change our resolution logic